checks/taskcluster/create_task.py

```diff
@@ -0,0 +1,78 @@
+"""
+A check to verify that tasks can be created.
+
+Information about the lastest created task is returned.
+"""
+import logging
+import textwrap
+from datetime import datetime, timedelta
+
+import taskcluster
+import taskcluster.aio
+import taskcluster.exceptions
+
+from poucave import config
+from poucave.typings import CheckResult
+
+from . import utils as tc_utils
+
+
+logger = logging.getLogger(__name__)
+
+
+TASK_METADATA = {
+    "owner": config.CONTACT_EMAIL,
+    "source": config.SOURCE_URL,
+    "description": textwrap.dedent(
+        """
+        This task is a test and is generated routinely by {config.SERVICE_NAME}
+        in order to monitor the taskcluster Queue services. It ensures that tasks
+        are able to be created, and they intentionally have a short expiry
+        to reduce resource usage.
+        """
+    ),
+}
+
+
+async def run(
+    root_url: str,
+    queue_id: str = "proj-taskcluster/gw-ci-ubuntu-18-04",
+    command: str = '/bin/bash -c echo "hola mundo!"',
```

Comment:
Wrapping /bin/bash wouldn't work on any worker I think

Reply:
Note, another option would be to not launch bash at all but instead use `/bin/echo` rather than the `echo` bash builtin, e.g.

```
    command: str = "/bin/echo 'hola mundo!'",
```

Reply:
oh thanks! I think I prefer this too